src/pyclass.rs

```diff
@@ -25,14 +24,10 @@ use std::{
 pub trait PyClass:
     PyTypeInfo<AsRefTarget = PyCell<Self>> + PyClassImpl<Layout = PyCell<Self>>
 {
+    /// Frozen or not
+    type Frozen: Frozen;
```

Comment:
Given that the `True` / `False` nature of this field exactly mirrors the macro attribute, I felt comfortable exposing this in the `PyClass` trait rather than the `PyClassImpl` trait. Also seems sensible given that users see this type in the public api docs for e.g. `PyRefMut`.

Reply:
(will merge this branch now, we can always change this before 0.16 if you disagree)